### DISCUSSION TITLE
Best VCTK speakers I've found

### DISCUSSION THREAD

**Discussion by T145** [coqui-ai/TTS#4044] [General]
Inspired by: https://github.com/coqui-ai/TTS/discussions/1891

https://github.com/T145/vctk-vits-speaker-comparison

I tried to use `tts_models/en/vctk/fast_pitch`, but the speaker IDs didn't work.

Has anyone had similar results and preferences? Or any that are completely different?

**Comment by eginhard** [comment]
The speakers of the VCTK Vits model are randomly shuffled, see #2258, so can't be compared with any other VCTK models.

**Reply by T145** [reply]
That's why I made the repo comparing the available speakers, due to them all being shuffled. Are you saying they all get shuffled every time the model is loaded?

**Reply by eginhard** [reply]
Ok, it wasn't clear to me from your post or that repo and because you wrote 

> I tried to use `tts_models/en/vctk/fast_pitch`, but the speaker IDs didn't work.
